src/ctap/storage.rs

```diff
@@ -59,11 +62,25 @@ const PIN_RETRIES: usize = 4;
 const ATTESTATION_PRIVATE_KEY: usize = 5;
 const ATTESTATION_CERTIFICATE: usize = 6;
 const AAGUID: usize = 7;
+#[cfg(not(feature = "with_ctap2_1"))]
 const NUM_TAGS: usize = 8;
+#[cfg(feature = "with_ctap2_1")]
+const MIN_PIN_LENGTH: usize = 8;
+#[cfg(feature = "with_ctap2_1")]
+const MIN_PIN_LENGTH_RP_IDS: usize = 9;
+#[cfg(feature = "with_ctap2_1")]
+const NUM_TAGS: usize = 10;
 
 const MAX_PIN_RETRIES: u8 = 6;
 const ATTESTATION_PRIVATE_KEY_LENGTH: usize = 32;
 const AAGUID_LENGTH: usize = 16;
+#[cfg(feature = "with_ctap2_1")]
+const DEFAULT_MIN_PIN_LENGTH: u8 = 4;
+#[cfg(feature = "with_ctap2_1")]
+const _DEFAULT_MIN_PIN_LENGTH_RP_IDS: Vec<String> = Vec::new();
```

Comment:
In a `no_std` context, `lazy_static` can be used with the `spin_no_std` feature, but that depends on `spin` which triggers a Cargo audit warning for us (https://github.com/google/OpenSK/issues/24).

Reply:
Is this "default min pin length RP ids" something specified in the standard? Do we really want to bake it in the binary? Or should we instead do as for the attestation certificate, aaguid, and friends and add a command to set it? We would have a store entry just for that default value (independent of the actual final value).

Reply:
It's probably the same use case, I agree. Since it is unused until I add the extension dealing with this value, can we agree on implementing that as part of that extension?

Reply:
Yes, I'm fine with that.